pkg/importer/spanner/sysl.arrai

```diff
@@ -0,0 +1,138 @@
+# size returns the size of an attribute
+let size = \length
+    cond {
+        length = 'MAX': '',
+        //seq.has_prefix('0x', length): '(' ++ //seq.trim_prefix('0x', length) ++ ')',
```

Comment:
Nit: in almost all cases, a string template is shorter _and_ easier to read that a string concatenation.

```suggestion
        //seq.has_prefix('0x', length): $`(${//seq.trim_prefix('0x', length)})`,
```

Reply:
ok